File: 33-ai-cost-monitoring/npm-package/monitoring.ts
import axios from 'axios';
const API = 'https://api.costkatana.com/api'\;
const KEY = process.env.COST_KATANA_API_KEY;

export async function getCostTrends() {
  const res = await axios.get(`${API}/monitoring/cost-trends`,
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('📈 Cost trends:', res.data.data);
  return res.data.data;
}

File: 33-ai-cost-monitoring/python-sdk/monitoring.py
import os, requests
API = "https://cost-katana-backend.store/api"
KEY = os.getenv("COST_KATANA_API_KEY")

def get_cost_trends():
    res = requests.get(f"{API}/monitoring/cost-trends",
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"📈 Cost trends: {res.json()['data']}")
    return res.json()["data"]
